java/viewshed-location/README.md

```diff
@@ -1,22 +1,36 @@
-# Viewshed Location
-A viewshed shows the visible and obstructed areas from an observer's vantage point. This sample demonstrates all of the configurable properties of a viewshed.
+# Viewshed location
 
-![Viewshed Location](viewshed-location.png)
+Perform a viewshed analysis from a defined vantage point. 
+
+![Image of viewshed location](viewshed-location.png)
+
+## Use case
+
+A 3D viewshed analysis is a type of visual analysis you can perform on a scene. The viewshed shows what can be seen from a given location. The output is an overlay with two different colors - one representing the visible areas (green) and the other representing the obstructed areas (red). Viewshed analysis is a form of "exploratory analysis", which means the results are calculated on the current scale of the data, and the results are generated very quickly. If more "conclusive" results are required, consider using a `GeoprocessingTask` to perform a viewshed instead.
 
 ## How to use the sample
-Use the seek bars on the bottom of the screen to change the properties of the viewshed and see them updated in real time. To move the viewshed, double touch and drag your finger across the screen. Lift your finger to stop moving the viewshed.
+
+Use the sliders to change the properties (heading, pitch, etc.), of the viewshed and see them updated in real time. To move the viewshed, double touch and drag your finger across the screen. Lift your finger to stop moving the viewshed.
 
 ## How it works
+
 1. Create a `LocationViewshed` passing in the observer location, heading, pitch, horizontal/vertical angles, and min/max distances.
-1. Set the property values on the viewshed instance for location, direction, range, and visibility properties. 
-1. Properties like `.setFrustumOutlineColor(<color>)` are set statically on `Viewshed`.
+2. Set the property values on the viewshed instance for location, direction, range, and visibility properties. 
```

Comment:
this sentence is a bit awkward. There are too many properties to simply give a code example, so maybe
```suggestion
2. Set the location, direction, range, and visibility properties of the viewshed instance.
```